sphinx_gallery/notebook.py

```diff
@@ -98,6 +99,18 @@ def rst2md(text):
 
     return text
 
+def rst2md(text):
+    """Converts the RST text from the examples docstrigs and comments
+    into markdown text for the Jupyter notebooks"""
```

Comment:
Will delegate the conversion to pypandoc if found in the system, otherwise will use our custom converter